_posts/2019-08-29-transform.md

```diff
@@ -74,29 +74,16 @@ layout: null
 <h1><a id="#sort">Sort</a></h1>
 <div id="wrapper">
     <div id="first"> 
-    <img src="/guide/images/transform_sort.png" width="100%"> 
+    <img src="{{'/guide/images/transform_sort.png'| relative_url}}" width="100%">
     </div>
     <div id="second">
     <h3>Description:</h3>
-    <p>Order table rows by an expression involving its variables.</p>
+    <p>Order table rows by an expression involving its variables. Use the checkbox for descending the order of rows.</p>
     <h3>Arguments</h3>
-    <p>The name of the column(s) to arrange the rows by -- seperated by commas.</p>
+    <p><b>Text Field: </b>The name of the column(s) to arrange the rows by -- seperated by commas.</p>
+    <p><b>Checkbox: </b>Check to sort table in descending order</p>
```

Comment:
Hm - just realized that there's no way to sort ascending by one column and descending by another, but how often does that ever come up?